### PR TITLE
Add wallet balances

### PR DESCRIPTION
**Pull request by guavajellyaaron**

### CODE REVIEW

File: src/express/handlers/jobs/minting.ts [hunk 1]
```diff
@@ -154,6 +154,14 @@ export const mintPaidSessionsHandler = async (req: express.Request, res: express
       });
     }
 
+    if (availableWallet.balance && availableWallet.balance < availableWallet.minBalance) {
+      Logger.log({ message: `${availableWallet.id} balance is lower than minimum balance`, event: 'mintPaidSessionsHandler.availableWallet.balance', category: LogCategory.NOTIFY });
+      return res.status(200).json({
+        error: false,
+        message: 'Not enough balance in wallet.'
```
Comment (cjkoepke) [inline src/express/handlers/jobs/minting.ts:161]: Wouldn't this technically be an "error"? I can see why not, but it also doesn't seem like it should be a "successful" response when the intent is to mint.

File: src/express/handlers/jobs/state/index.ts [hunk 2]
```diff
@@ -17,13 +18,15 @@ interface StateResponseBody {
 export const stateHandler = async (req: express.Request, res: express.Response) => {
   try {
     const accessQueueSize = await AccessQueues.getAccessQueueCount();
-    const mintingQueueSize = await (await ActiveSessions.getPaidPendingSessions({ limit: 20000 })).length;
+    const mintingQueueSize = (await ActiveSessions.getPaidPendingSessions({ limit: 20000 })).length;
```
Comment (cjkoepke) [inline src/express/handlers/jobs/state/index.ts:21]: Maybe gracefully access `?.length` or at least check for possible issues when `length` isn't present.

Comment (guavajellyaaron) [reply src/express/handlers/jobs/state/index.ts:21]: You still have to await it unfortunately. And I believe, the response will always be an array.

File: src/express/handlers/jobs/minting.ts (hunk 1 above)
Comment (guavajellyaaron) [reply src/express/handlers/jobs/minting.ts:161]: Good call. Maybe a 403 forbidden or something like that

**Pull request merged**